pkgs/by-name/ma/materialx/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  darwin,
+  libX11,
+  libXt,
+  libGL,
+  openimageio,
+  imath,
+  python3Packages,
+  python3
```

I've seen a preference for formatting package attributes with a leading comma:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake
, darwin
, libX11
, libXt
, libGL
, openimageio
, imath
, python3Packages
, python3
```

---

actually this is just formated using the `nixfmt` formatter. Do you have a source/RFC/PR which suggest one formatter over the other?

---

Thank you for calling this out, @gador. I was not aware of the preference change through `nixfmt`, please discard my suggestion.